src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -33,12 +34,13 @@
 _ADDRESS_SPACE_LIMIT = 4096
 _TIMEOUT_PER_INPUT_FUZZ = 25
 _TIMEOUT_PER_INPUT_REPR = 60
-_DEFAULT_ARGUMENTS = [
-    '--exit_on_crash=1',
-    f'--fork_server={_SERVER_COUNT}',
-    f'--rss_limit_mb={_RSS_LIMIT}',
-    f'--address_space_limit_mb={_ADDRESS_SPACE_LIMIT}',
-]
+_DEFAULT_ARGUMENTS = {
+    'exit_on_crash': 1,
```

Comment:
This line seems redundant, given that it will be unconditionally skipped below.

Reply:
Good catch!